res/values/strings.xml

```diff
@@ -447,8 +455,6 @@
     <string name="import_backup_no_backup_found">No backups found.\n\nCopy the backup to \"%1$s\" and try again. Alternatively, press \"Start messaging\" to continue with the normal setup process.</string>
     <!-- Translators: %1$s will be replaced by the e-mail address -->
     <string name="login_error_cannot_login">Cannot login as \"%1$s\". Please check if the e-mail address and the password are correct.</string>
-    <!-- Translators: %1$s will be replaced by the server name (eg. imap.somewhere.org) and %2$s will be replaced by the human-readable response from the server. this response may be a single word or some sentences and may or may not be localized. -->
-    <string name="login_error_server_response">Response from %1$s: %2$s\n\nSome providers may e-mail you additional information about this problem; you should be able to check your inbox using your regular e-mail app or web site. Consult your provider or friends if you run into problems.</string>
```

Comment:
this one seems to be still used in the core on iOS (DC_STR_SERVER_RESPONSE). I didn't follow the core development, so maybe it's already deprecated.

Reply:
Well it's deprecated with the introduction of the connectivity-view, as error_network's are removed. But I can leave it in for some time if you say it's better

Reply:
Let's leave it in for now, then